faqs/galaxy/histories_download_entire_histories.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Download entire histories
+area: histories
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+1. Click on the gear icon {% icon galaxy-gear %} on the top of the history panel
+2. Select "Export to File" from the History menu
+3. Click on the link generated
```

Comment:
This step can be expanded a bit. They first need to click a link to start the process, then wait, then click on a new link to download the history itself.

If possible, try to run through the steps of these FAQs yourself, and see if the instructions can be improved (sometimes they are outdated, or not formulated in a good way for newcomers, so your opinions on the quality of the instructions here is quite useful to us as well!)